flow360/component/flow360_params/solvers.py

```diff
@@ -284,11 +323,58 @@ class HeatEquationSolver(Flow360BaseModel):
                     absoluteTolerance=1e-10
             )
         )
-
     """
 
-    model_type: Literal["HeatEquation"] = pd.Field("HeatEquation", alias="modelType", const=True)
+    model_type: Literal["HeatEquation"] = pd.Field("HeatEquation", alias="modelType")
+    linear_solver_config: Optional[LinearSolver] = pd.Field(
+        alias="linearSolverConfig", default=LinearSolver()
+    )
+    order_of_accuracy: Optional[Literal[1, 2]] = pd.Field(alias="orderOfAccuracy")
+    CFL_multiplier: Optional[PositiveFloat] = pd.Field(alias="CFLMultiplier")
+    update_jacobian_frequency: Optional[PositiveInt] = pd.Field(alias="updateJacobianFrequency")
+    max_force_jac_update_physical_steps: Optional[NonNegativeInt] = pd.Field(
+        alias="maxForceJacUpdatePhysicalSteps"
+    )
     equation_eval_frequency: Optional[PositiveInt] = pd.Field(alias="equationEvalFrequency")
+
+
+class TransitionModelSolver(GenericFlowSolverSettings):
+    """:class:`TransitionModelSolver` class for setting up transition model solver
+
+    Parameters
+    ----------
+
+    (...)
+
+    Returns
+    -------
+    :class:`TransitionModelSolver`
+        An instance of the component class TransitionModelSolver.
+
+    Example
+    -------
+    >>> ts = TransitionModelSolver(absolute_tolerance=1e-10)
+    """
+
+    model_type: Optional[Literal["AmplificationFactorTransport"]] = pd.Field(
```

Comment:
we should also support modelType=None. It can be generic none model solver also used in turbulence model

Reply:
Fixed in c3c40f4